packages/protocol/contracts/team/airdrop/ERC20Airdrop.sol

```diff
@@ -41,8 +41,19 @@ contract ERC20Airdrop is MerkleClaimable {
         vault = _vault;
     }
 
-    function _claimWithData(bytes calldata data) internal override {
+    function _claimWithData(
+        bytes calldata data,
+        address delegatee,
+        uint256 nonce,
+        uint256 expiry,
+        uint8 v,
+        bytes32 r,
+        bytes32 s
+    ) internal override {
```

Comment:
If both ERC20Airdrop and ERC20Airdrop2 allow claiming, we shall create a parent contract and move shared code there.